RecoLocalTracker/SiPixelRecHits/src/PixelCPEGeneric.cc

```diff
@@ -744,3 +744,22 @@ LocalError PixelCPEGeneric::localError(DetParam const& theDetParam, ClusterParam
 
   return LocalError(xerr_sq, 0, yerr_sq);
 }
+
+void PixelCPEGeneric::fillPSetDescription(edm::ParameterSetDescription& desc) {
+  desc.add<double>("eff_charge_cut_highX", 1.0);
+  desc.add<double>("eff_charge_cut_highY", 1.0);
+  desc.add<double>("eff_charge_cut_lowX", 0.0);
+  desc.add<double>("eff_charge_cut_lowY", 0.0);
+  desc.add<double>("size_cutX", 3.0);
+  desc.add<double>("size_cutY", 3.0);
+  desc.add<double>("EdgeClusterErrorX", 50.0);
+  desc.add<double>("EdgeClusterErrorY", 85.0);
+  desc.add<bool>("inflate_errors", false);
+  desc.add<bool>("inflate_all_errors_no_trk_angle", false);
+  desc.add<bool>("UseErrorsFromTemplates", true);
+  desc.add<bool>("TruncatePixelCharge", true);
+  desc.add<bool>("IrradiationBiasCorrection", false);
+  desc.add<bool>("DoCosmics", false);
```

Comment:
what is not needed?

Reply:
Sorry @mmusich , the comment shouldn't have been attached here but to the same parameter in the fillDescription of PixelCPETemplateReco below, where you write "needed because it's in the HLT configuration".
I'm moving my comment in the right place

Reply:
closing this conversion, to follow it up in https://github.com/cms-sw/cmssw/pull/27694#discussion_r315722802